gluten-core/src/main/scala/io/glutenproject/execution/ExpandExecTransformer.scala

```diff
@@ -207,51 +208,29 @@ case class ExpandExecTransformer(projections: Seq[Seq[Expression]],
     }
   }
 
-  override def doValidateInternal(): Boolean = {
+  override protected def doValidateInternal(): ValidationResult = {
     if (!BackendsApiManager.getSettings.supportExpandExec()) {
-      this.appendValidateLog(
-        s"Validation failed for ${this.getClass.toString}" +
-          s" due to: {ExpandExec}. ")
-      return false
+      return notOk("backend does not support expand")
```

Comment:
will this message like `backend does not support expand` be shown in Spark UI directly or you will further process it? I think we'd better consider both meaning and simplicity.

Reply:
See the pr description, this pr would print something like:

```
23/07/03 20:22:54 WARN GlutenFallbackReporter: Validation failed for plan: Scan parquet default.t, due to: Some(columnar FileScan is not enabled in FileSourceScanExec).
```

This logging format is based on your previous pr. We can improve it in UI in next pr if we want.

Reply:
Yes. My previous planned next step is to format the error message further. Eventually, we should only show some key words for failed operations like "data type: map, function: get_struct_feild(XXXX)".